core/src/main/java/org/apache/accumulo/core/clientImpl/mapreduce/lib/ConfiguratorBase.java

```diff
@@ -284,16 +287,17 @@ public static AuthenticationToken getTokenFromFile(Configuration conf, String pr
       String tokenFile) {
     FSDataInputStream in = null;
     try {
-      URI[] uris = DistributedCacheHelper.getCacheFiles(conf);
-      Path path = null;
-      for (URI u : uris) {
-        if (u.toString().equals(tokenFile)) {
-          path = new Path(u);
-        }
+      Path path;
+      // See if the "tokenfile" symlink was created and try to open the file it points to by it.
+      File tempFile = new File(ConfiguratorBase.cachedFileName);
+      if (tempFile.exists()) {
+        path = new Path(ConfiguratorBase.cachedFileName);
```

Comment:
Another conversion of local File to HDFS Path, which I'm not sure will work correctly.